cli/src/program.rs

```diff
@@ -2178,16 +2181,29 @@ fn send_deploy_messages(
                     write_messages,
                     &[payer_signer, write_signer],
                 ),
-                ConnectionCache::Quic(cache) => TpuClient::new_with_connection_cache(
-                    rpc_client.clone(),
-                    &config.websocket_url,
-                    TpuClientConfig::default(),
-                    cache,
-                )?
-                .send_and_confirm_messages_with_spinner(
-                    write_messages,
-                    &[payer_signer, write_signer],
-                ),
+                ConnectionCache::Quic(cache) => {
+                    let tpu_client_fut = solana_client::nonblocking::tpu_client::TpuClient::new_with_connection_cache(
+                        rpc_client.get_inner_client().clone(),
+                        config.websocket_url.as_str(),
+                        solana_client::tpu_client::TpuClientConfig::default(),
+                        cache,
+                    );
+                    let tpu_client = rpc_client
+                        .runtime()
+                        .block_on(tpu_client_fut)
+                        .expect("Should return a valid tpu client");
+
+                    send_and_confirm_transactions_in_parallel_blocking(
+                        rpc_client.clone(),
+                        Some(tpu_client),
+                        write_messages,
+                        &[payer_signer, write_signer],
+                        SendAndConfrimConfig {
+                            resign_txs_count: Some(5),
```

Comment:
maybe expose as a cli arg `--transaction-resign-attempts NUM_ATTEMPTS` default `0` for saftety?

Reply:
I will update this in the other PR, for this PR it is better that we have the same number of resign attempts like the previous implementation.